### ISSUE TITLE
Blanket rm statements will lead to user files being deleted

### ISSUE THREAD

**Issue by jangliss** [km4ack/pi-build#101]
Lines 488 to 492 need to be looked at.  I download files to my Downloads folder, the last step here is to delete everything in that folder.  

Other cleanup stuff here might need to be weighed in on user preference, such as the "apt -y autoremove", I might have packages listed as unused, but still be using them, the package manager just might not know it (for example libraries I installed using apt and then manually built against).

**Comment by km4ack** [comment]
Good catch. This was left over from version 1 & 2 which were only designed to be run on a fresh build. Labeling as a bug to move up the priority list.

**Comment by km4ack** [comment]
Slated for 3.0.4 release
